src/2048-expert/index.js

```diff
@@ -1,8 +1,12 @@
 document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Are you able to remove this listener wrapper and add `type="module"` to the script elements in `index.html`?
`<script type="module" src="../main.js"></script>` can also be moved to where the other script elements are. 
![image](https://user-images.githubusercontent.com/78716153/184763247-e1b1ff94-5bc4-4b80-8905-66ac82e8bb67.png)